backend/populateDB/schema.sql

```diff
@@ -659,22 +668,24 @@ $$ language plpgsql strict security definer;
 
 comment on function floods.delete_status_duration(integer) is 'Deletes a status duration.';
 
--- Create function to create new communities
+-- comment on function floods.new_community(text, text) is 'Adds a community.';
```

Comment:
this kinda makes me curious about how its going to work for a user to upload a shape from the front end. Might we need to be ready to run a conversion from shapefile to geojson? Probably thinking too far ahead.